uiowa/uiowa
# Events button hotfix

Resolves #5745 




# How to test



See
https://international.uiowa.edu/
https://international.uiowa.edu/ISSS
https://international.uiowa.edu/study-abroad
lacking the "view more events" buttons.

Checkout and pull
`blt ds --site=international.uiowa.edu`
Visit the pages locally, and see that the "view more events" buttons are now visible again.

Sync a V2 site (eg hawkeyemarchingband.uiowa.edu)
Add an events block w/view more button, and see that it appears
